src/cephadm/cephadm.py

```diff
@@ -9984,6 +9984,24 @@ def cephadm_init_ctx(args: List[str]) -> CephadmContext:
     return ctx
 
 
+def check_required_global_args(ctx: CephadmContext, args: List[str]) -> None:
```

Comment:
Feels a smidge like overkill to me. I think it would be simpler to add an
```python
if not ctx.image:
    raise ValueError("the inspect-image requires the --image parameter")
    # feel free to keep the note. I'm lazy and didn't want to copy it.
```

If the issue is that the decorator needs it, then another decorator could solve the problem:
```python
def require_image(f):
   def _require_image(ctx):
       if not ctx.image:
           # ..etc..
       return f(...)
    return _require_image
```

To me this seems like a lot less generic code to think about.

Reply:
agreed, the decorator is much cleaner. Moved over to that.